diff -up conf/nginx.conf.orig conf/nginx.conf
--- conf/nginx.conf.orig	2009-06-14 00:10:47.579143985 +0700
+++ conf/nginx.conf	2009-06-14 00:19:20.261143691 +0700
@@ -1,57 +1,69 @@
+user  nginx;
+worker_processes  10;
+worker_rlimit_nofile 100000;
+
+error_log   /var/log/nginx/error.log;
+#error_log  /var/log/nginx/error.log  notice;
+#error_log  /var/log/nginx/error.log  info;
 
-#user  nobody;
-worker_processes  1;
-
-#error_log  logs/error.log;
-#error_log  logs/error.log  notice;
-#error_log  logs/error.log  info;
-
-#pid        logs/nginx.pid;
+pid        /var/run/nginx.pid;
 
 
 events {
     worker_connections  1024;
+    use epoll;
 }
 
 
 http {
-    include       mime.types;
+    include       /etc/nginx/mime.types;
     default_type  application/octet-stream;
 
-    #log_format  main  '$remote_addr - $remote_user [$time_local] "$request" '
-    #                  '$status $body_bytes_sent "$http_referer" '
-    #                  '"$http_user_agent" "$http_x_forwarded_for"';
+    log_format  main  '$remote_addr - $remote_user [$time_local] "$request" '
+                      '$status $body_bytes_sent "$http_referer" '
+                      '"$http_user_agent" "$http_x_forwarded_for"';
 
-    #access_log  logs/access.log  main;
+    access_log  /var/log/nginx/access.log  main;
 
     sendfile        on;
-    #tcp_nopush     on;
-
-    #keepalive_timeout  0;
+    tcp_nopush      on;
+    tcp_nodelay     on;
+    server_tokens   off;
+    gzip            on;
+    gzip_static     on;
+    gzip_comp_level 5;
+    gzip_min_length 1024;
     keepalive_timeout  65;
+    limit_zone   myzone  $binary_remote_addr  10m;
 
-    #gzip  on;
+    # Load config files from the /etc/nginx/conf.d directory
+    include /etc/nginx/conf.d/*.conf;
 
     server {
+        limit_conn   myzone  10;
         listen       80;
-        server_name  localhost;
+        server_name  _;
 
         #charset koi8-r;
 
         #access_log  logs/host.access.log  main;
 
         location / {
-            root   html;
+            root   /usr/share/nginx/html;
             index  index.html index.htm;
         }
 
-        #error_page  404              /404.html;
+        error_page  404              /404.html;
+
+        location = /404.html {
+            root   /usr/share/nginx/html;
+        }
 
         # redirect server error pages to the static page /50x.html
         #
         error_page   500 502 503 504  /50x.html;
         location = /50x.html {
-            root   html;
+            root   /usr/share/nginx/html;
         }
 
         # proxy the PHP scripts to Apache listening on 127.0.0.1:80
